sublime_linter.py

```diff
@@ -234,7 +223,7 @@ def lint_all_views():
     for window in sublime.windows():
         for view in window.views():
             if view.buffer_id() in persist.view_linters:
-                hit(view)
+                hit(view, "on_load")
```

Comment:
This one only gets called when the config changes. (Which in itself is not optimized yet #1312) And we want 'on_user_request' here.